packages/ember-glimmer/lib/component-managers/capabilities.ts

```diff
@@ -0,0 +1,30 @@
+import { assert } from 'ember-debug';
+import { assign } from 'ember-utils';
+
+// because we're not yet compatible with latest Glimmer
+// we lack typings in Ember.js.
+// steal capabilities interface from http://bit.ly/2gMdhv5 for now
+export interface ComponentCapabilities {
+  dynamicLayout: boolean;
+  dynamicTag: boolean;
+  prepareArgs: boolean;
+  createArgs: boolean;
+  attributeHook: boolean;
+  elementHook: boolean;
+}
+
+// default capabilities values stolen from http://bit.ly/2zfZQyz
+export const DEFAULT_CAPABILITIES: ComponentCapabilities = {
```

Comment:
@wycats @chadhietala could you guys confirm that these values are correct default values?

Reply:
If these are for Ember Components then I believe you need `attributeHook`. If I remember correctly this is how we do attribute bindings.